packages/odata-service-inquirer/src/types.ts

```diff
@@ -262,6 +262,12 @@ export type SystemSelectionPromptOptions = {
      *
      */
     defaultChoice?: string;
+    /**
+     * Only show the default choice in the system selection prompt, this is used to skip the system selection prompt if the default choice is already known.
+     * If the `defaultChoice` value is not found in the systems choices, or the `defaultChoice` option is not specified,
+     * this option will not be applied and the full list of choices will be presented o the user.
```

Comment:
```suggestion
     * this option will not be applied and the full list of choices will be presented to the user.
```

Reply:
Fixed in: beb38a9e9680bd34b088cf8ba5db0628e3e70c0d